src/App.js

```diff
@@ -1,6 +1,16 @@
 import React from 'react';
 import './App.css';
+import { Pagination } from './components/Pagination';
 
-const App = () => <h1>Pagination</h1>;
+const App = () => (
+  <>
+    <h1>Pagination</h1>
+    <Pagination
```

Comment:
4. Add an optional param withInfo to show extra info before the buttons (6 - 10 of 20)